catalog/core/catalog-core-api/src/main/java/ddf/catalog/event/Subscription.java

```diff
@@ -36,4 +36,8 @@
      */
     public DeliveryMethod getDeliveryMethod();
 
+    public boolean isActive();
```

Comment:
If you're keeping both of these methods, you'll want to javadoc them to indicate the difference between expected behavior and performance. If pinging is expensive and should only be run periodically vs a simple field check, you'll want to indicate here that is the case; furthermore, you'll want to indicate to future implementers that difference so they don't use a different approach that switch the complexity or something similar.